tests/pipelines/ner/test_disorders/test_all.py

```diff
@@ -0,0 +1,108 @@
+import pytest
```

Comment:
Could we remove the `test_` prefrix from `test_{specific_entity}` (or replace it), to reserve it for files that contain runnable tests ?

Reply:
Agree !